node/core/backing/src/lib.rs

```diff
@@ -298,21 +323,272 @@ fn table_attested_to_backed(
 	})
 }
 
+async fn store_available_data(
+	tx_from: &mut mpsc::Sender<FromJob>,
+	id: Option<ValidatorIndex>,
+	n_validators: u32,
+	candidate_hash: CandidateHash,
+	available_data: AvailableData,
+) -> Result<(), Error> {
+	let (tx, rx) = oneshot::channel();
+	tx_from.send(FromJob::AvailabilityStore(
+			AvailabilityStoreMessage::StoreAvailableData(
+				candidate_hash,
+				id,
+				n_validators,
+				available_data,
+				tx,
+			)
+		)
+	).await?;
+
+	let _ = rx.await?;
+
+	Ok(())
+}
+
+// Make a `PoV` available.
+//
+// This will compute the erasure root internally and compare it to the expected erasure root.
+// This returns `Err()` iff there is an internal error. Otherwise, it returns either `Ok(Ok(()))` or `Ok(Err(_))`.
```

Comment:
It's not obvious why we don't add an `InvalidErasureRoot` variant to `Error` and flatten the output here.

Reply:
This error doesn't need to be exposed to the higher level, which is what `Error` communicates.